feature/gnoi/system/tests/ping_test/ping_test.go

```diff
@@ -435,17 +431,21 @@ func TestGNOIPing(t *testing.T) {
 				if responses[i].Source != tc.expectedReply.Source {
 					t.Errorf("Ping reply source: got %v, want %v", responses[i].Source, tc.expectedReply.Source)
 				}
-				if responses[i].Time < tc.expectedReply.Time {
-					t.Errorf("Ping time: got %v, want >= %v", responses[i].Time, tc.expectedReply.Time)
-				}
-				if responses[i].Bytes < tc.expectedReply.Bytes {
-					t.Errorf("Ping Bytes: got %v, want >= %v", responses[i].Bytes, tc.expectedReply.Bytes)
-				}
-				if responses[i].Sequence < tc.expectedReply.Sequence {
-					t.Errorf("Ping time: got %v, want >= %v", responses[i].Sequence, tc.expectedReply.Sequence)
-				}
-				if responses[i].Ttl < tc.expectedReply.Ttl {
-					t.Errorf("Ping TTL: got %v, want >= %v", responses[i].Ttl, tc.expectedReply.Ttl)
+
+				if i < len(responses)-1 {
```

Comment:
This doesn't look great. I feel we should make the top level if statement to `len(responses)-1` and then add the logic for last element outside the loop.
e.g.
```
for i := 0; i < len(responses)-1; i++ {
// log echo response data
}
summary := responses[len(responses)-1]
// log summary response data